tests/lib/tools/fs-state

```diff
@@ -1,38 +1,53 @@
 #!/bin/bash
 
+show_help() {
+	echo "usage: fs-state wait-for-file [filename] [iterations] [sleep-time]"
+	echo "usage: fs-state ensure-dir-exists [direname]"
+	echo "usage: fs-state ensure-dir-exists-backup-real [direname]"
+	echo "usage: fs-state clean-dir [direname]"
+	echo "usage: fs-state ensure-file-exists [filename]"
+	echo "usage: fs-state clean-file [filename]"
+}
+
 wait_for_file() {
-    the_file="$1"
-    iters="$2"
-    sleep_time="$3"
+    local file="$1"
+    local iters="$2"
+    local sleep_time="$3"
 
     for _ in $(seq "$iters"); do
-        if [ -e "$the_file" ]; then
+        if [ -e "$file" ]; then
+        	echo "File exists: $file"
             return 0
         fi
         sleep "$sleep_time"
     done
+    echo "File does not exists: $file"
     return 1
 }
 
 ensure_dir_exists() {
-    dir="$1"
+    local dir="$1"
     if ! [ -d "$dir" ]; then
+    	echo "Creating dir: $dir"
```

Comment:
Space misalignment. Can you please check if tabs and spaces are consistently used here.

Reply:
thanks